### PR TITLE
Move backend from python to .NET

### PR DESCRIPTION
**Pull request by anetteu**

### CODE REVIEW

**Review by GodVenn — APPROVED** [review]
LGTM

**Review by aestene — CHANGES_REQUESTED** [review]
Could you remove the python package build workflow file?

**Review by aestene — APPROVED** [review]

File: .github/workflows/backend_lint_and_test.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+name: Backend
+
+on:
+  push:
+    branches:
+      - dotnet
+  pull_request:
+    branches:
+      - dotnet
```
Comment (GodVenn) [inline .github/workflows/backend_lint_and_test.yml:9]:
```suggestion
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```

**Review by GodVenn — CHANGES_REQUESTED** [review]
Please also change pipelines to trigger on correct branch

File: .github/workflows/backend_lint_and_test.yml (hunk 1 above)
Comment (MortFred) [reply .github/workflows/backend_lint_and_test.yml:9]: Good catch!

**Review by GodVenn — APPROVED** [review]
LeGiTiMate

**Pull request merged**